modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java

```diff
@@ -203,6 +211,63 @@ public void setDiscriminator(CodegenDiscriminator discriminator) {
         this.discriminator = discriminator;
     }
 
+    /**
+     * Returns a list of allowed discriminator mapping values for this schema object.
+     * 
+     * The list of all possible schema discriminator mapping values is obtained
+     * from explicit discriminator mapping values in the OpenAPI document, and from
+     * inherited discriminators through oneOf, allOf, anyOf.
+     * For example, a discriminator may be defined in a 'Pet' schema as shown below.
+     * The Dog and Cat schemas inherit the discriminator through the allOf reference.
+     * In the 'Pet' schema, the supported discriminator mapping values for the
+     * 'objectType' properties are 'Dog' and 'Cat'.
+     * The allowed discriminator mapping value for the Dog schema is 'Dog'.
+     * The allowed discriminator mapping value for the Cat schema is 'Dog'.
+     * 
+     * Pet:
+     *   type: object
+     *   discriminator:
+     *     propertyName: objectType
+     *   required:
+     *     - objectType
+     *   properties:
+     *     objectType:
+     *     type: string
+     * Dog:
+     *   allOf:
+     *   - $ref: '#/components/schemas/Pet'
+     *   - type: object
+     *     properties:
+     *       p1:
+     *         type: string
+     * Cat:
+     *   allOf:
+     *   - $ref: '#/components/schemas/Pet'
+     *   - type: object
+     *     properties:
+     *       p2:
+     *         type: string
+     * 
+     * @return the list of allowed discriminator mapping values.
+     */
+    public List<String> getAllowedDiscriminatorMappingValues() {
+        List<String> mappingValues = new ArrayList<String>();
+        for (CodegenDiscriminator.MappedModel mm : discriminator.getMappedModels()) {
+            if (name.equals(mm.getModelName())) {
+                mappingValues.add(mm.getMappingName());
+            }
+        }     
+        return mappingValues;   
+    }
```

Comment:
Where are we using this? I can't find a call site.
If we aren't using it, why include it?
Our users can already iterate through discriminator MappedModel.
If we want to provide an easier way to do this, why not put it in the CodegenDiscriminator class instead?

Reply:
I am planning to remove it, after your PR has been merged. I accidentally cherry picked some commits I didn't want to merge